src/erlavro.app.src

```diff
@@ -5,7 +5,9 @@
   {registered, []},
   {applications, [
                   kernel,
-                  stdlib
+                  stdlib,
+                  mochijson3,
```

Comment:
Perhaps put them in 'included_applications'? They are static ones, don't need to be started.